src/Context/PaymentContext.tsx

```diff
@@ -0,0 +1,108 @@
+import { createContext, useContext, useEffect, useMemo, useState } from "react";
+import { PaymentContextType, PaymentDetails } from "../Types/ProductsTypes";
+import { get, post } from "../utils/axios";
+import { v4 as uuid } from "uuid";
+
+const defaultState: PaymentContextType = {
+  paymentInfo: [],
+};
+
+const localStorageData = localStorage.getItem("PaymentState");
+const initialState: PaymentContextType = localStorageData
+  ? JSON.parse(localStorageData)
+  : defaultState;
+
+export const PaymentContext = createContext<PaymentContextType>(initialState);
+
+export const PaymentContextProvider = ({
+  children,
+}: {
+  children: React.ReactNode;
+}) => {
+  const [uState, setUState] = useState<PaymentContextType>(initialState);
+
+  const addPayment = async (pData: PaymentDetails) => {
+    const {
+      data: { key },
+    } = await get("http://localhost:4000/api/getkey");
```

Comment:
we should create axios instance and add base URL there, so no need to add base URL every time

Reply:
i added base url but for payment service cuurently i am using diffrent server so i used diffrent url